sphinxcontrib_verilog_diagrams/__init__.py

```diff
@@ -268,7 +281,7 @@ def diagram_netlistsvg(ipath, opath, module='top', flatten=False):
     run_yosys(
         src=ipath,
         cmd = """\
-prep -top {top} {flatten}; cd {top}; write_json {ojson}
+prep -top {top} {flatten}; cd {top}; write_json -compat-int {ojson}
```

Comment:
The compatibility mode seems to work incorectly when added to the standard conda Yosys version.